modules/metrics/src/main/java/com/spotify/apollo/metrics/semantic/SemanticServiceMetrics.java

```diff
@@ -191,6 +197,31 @@ private CachedMeters metersForEndpoint(String endpoint) {
            Optional.empty();
   }
 
+  /**
+   * Checks for latency-slo-metrics configuration options and sets up
+   * metrics that track how many requests meet a certain latency goal.
+   *
+   * @param id
+   * @param endpoint
+   * @return
+   */
+  private Optional<LatencySloTracker> requestLatencySloTracker(MetricId id, String endpoint) {
+    final String[] endpointComponents = endpoint.split(":");
+    final String key = String.format("latency-slo-metrics.%s", endpointComponents[1],
+        endpointComponents[0]);
+    final String allKey = "latency-slo-metrics.all-endpoints";
+    final boolean hasEndpointKey = config.hasPath(key);
+    final boolean hasAllKey = config.hasPath(allKey);
+    if (hasEndpointKey || hasAllKey) {
+      final int threshold = hasEndpointKey ? config.getInt(key) : config.getInt(allKey);
```

Comment:
@jsferrei Does this mean it would be possible to add something like this to the conf?
 
```
latency-slo-metrics.endpoint1: 50
latency-slo-metrics.endpoint2: 150
latency-slo-metrics.all-endpoints: 200
```

Reply:
@frecon Yes

Reply:
maybe the threshold should be included in one of the tags of the metric? I'm thinking about what happens if I change the threshold in the configuration, and then the reported metrics look very different

Reply:
@mattnworb good call

Reply:
@jsferrei that config example is missing the method (?) what happens if you leave the method off.

Reply:
Method is required and will always be passed in as part of `endpoint`, I believe. The above example should include method.